packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -177,11 +143,9 @@ test.describe('PieCookieBanner - Component tests', () => {
         expect(isNecessaryPreferenceToggleDisabled).toBe(true);
     });
 
-    test('should toggle all preferences if the `all` preference node is set to true', async ({ mount }) => {
+    test('should toggle all preferences if the `all` preference node is set to true', async () => {
         // Arrange
-        await mount(PieCookieBanner, {
-            props: {} as CookieBannerProps,
-        });
+        await pieCookieBannerComponent.load({ defaultPreferences: null });
 
         // Act
         await pieCookieBannerComponent.clickManagePreferencesAction();
```

Comment:
Candidate for a future PR, do you think it would be worth wrapping this logic that gets all the preferences?